### PR TITLE
Update Docs Around Callset Cleanup and Cost [VS-1107]

### PR DESCRIPTION
**Pull request by rsasch**

### CODE REVIEW

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -88,13 +88,26 @@
 1. `GvsCallsetCost` workflow
     - This workflow calculates the total BigQuery cost of generating this callset (which is not represented in the Terra UI total workflow cost) using the above GVS workflows; it's used to calculate the cost as a whole and by sample.
 
+
+## Internal sign-off protocol
+
+The Variants team currently has the following VDS internal sign-off protocol:
+
+1. Generate a VDS for the candidate callset into the "delivery" bucket.
+1. Open up the VDS in a beefy notebook and confirm the "shape" looks right.
```
[the remark this answers is not in the document]
Comment (rsasch) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md]: good idea — thanks!

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 2]
```diff
@@ -88,13 +88,26 @@
...
 
+
+## Internal sign-off protocol
+
+The Variants team currently has the following VDS internal sign-off protocol:
+
+1. Generate a VDS for the candidate callset into the "delivery" bucket.
+1. Open up the VDS in a [beefy](vds/cluster/AoU%20VDS%20Cluster%20Configuration.md) notebook and confirm the "shape" looks right.
+1. Run `GvsPrepareRangesCallset.wdl` to generate a prepare table of VET data
+1. Run `GvsCallsetStatistics.wdl` to generate callset statistics for the candidate callset using the prepare VET created in the preceding step
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:99]:
```suggestion
1. Run `GvsCallsetStatistics.wdl` to generate callset statistics for the candidate callset using the prepare VET table created in the preceding step
```

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 3]
```diff
@@ -88,13 +88,26 @@
...
+
+
 ## Main Deliverables (via email to stakeholders once the above steps are complete)
 
 The Callset Stats and S&P files can be simply `gsutil cp`ed to the AoU delivery bucket since they are so much smaller.
 1. GCS location of the VDS in the AoU delivery bucket
-2. Fully qualified name of the BigQuery dataset (composed of the `project_id` and `dataset_name` inputs from the workflows)
-3. GCS location of the CSV output from `GvsCallsetStatistics` workflow in the AoU delivery bucket
-4. GCS location of the TSV output from `GvsCalculatePrecisionAndSensitivity` in the AoU delivery bucket
+1.Fully qualified name of the BigQuery dataset (composed of the `project_id` and `dataset_name` inputs from the workflows)
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:108]:
```suggestion
1. Fully qualified name of the BigQuery dataset (composed of the `project_id` and `dataset_name` inputs from the workflows)
```

File: scripts/variantstore/docs/aou/cleanup/cleanup.md [hunk 4]
```diff
@@ -1,117 +1,39 @@
...
 
-As a general rule, any artifacts that have clearly become obsolete (e.g. VDSes with known issues that have been
-superseded by corrected versions, obsolete sets of prepare tables, etc.) should be deleted ASAP. If it's not clear to
-the Variants team whether an artifact should be cleaned up or not, we should calculate the monthly cost to preserve the
-artifact (e.g. the sum of all relevant GCS or BigQuery storage costs) as well as the cost to regenerate the artifact.
-Reach out to Lee with these numbers for his verdict on whether to keep or delete.
+As a general rule, any artifacts that have clearly become obsolete (e.g. VDSes with known issues that have been  superseded by corrected versions, obsolete sets of prepare tables, etc.) should be deleted ASAP. If it's not clear to the Variants team whether an artifact should be cleaned up or not, [we should calculate the monthly cost to preserve the artifact (e.g. the sum of all relevant GCS or BigQuery storage costs) as well as the cost to regenerate the artifact](Cost.md).
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/cleanup/cleanup.md:7]:
```suggestion
As a general rule, any artifacts that have clearly become obsolete (e.g. VDSes with known issues that have been superseded by corrected versions, obsolete sets of prepare tables, etc.) should be deleted ASAP. If it's not clear to the Variants team whether an artifact should be cleaned up or not, [we should calculate the monthly cost to preserve the artifact (e.g. the sum of all relevant GCS or BigQuery storage costs) as well as the cost to regenerate the artifact](Cost.md).
```

File: scripts/variantstore/docs/aou/cleanup/cleanup.md [hunk 5]
```diff
@@ -1,117 +1,39 @@
...
+
+Reach out to leadership with these numbers for his verdict on whether to keep or delete.
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/cleanup/cleanup.md:9]:
```suggestion
Reach out to leadership with these numbers for a verdict on whether to keep or delete.
```

**Review by gbggrant — APPROVED** [review]
Looks good. A couple of nits. Take 'em or leave 'em.

**Pull request merged**